python2verilog/backend/verilog/codegen.py

```diff
@@ -314,13 +314,13 @@ def get_module_str(self):
         """
         return self.get_module_lines().to_string()
 
-    def get_testbench(self, random_wait: bool = False):
+    def get_testbench(self, random_ready: bool):
         """
         Creates testbench with multiple test cases
 
         Each element of self.context.test_cases represents a single test case
 
-        :param random_wait: whether or not to have random _wait signal in the while loop
+        :param random_ready: whether or not to have random ready signal in the while loop
```

Comment:
No need for the `:` at the front of `param`

Reply:
this is the format sphinx expects.